File: moneylog_spring/src/main/java/bitc/fullstack503/moneylog_spring/dto/TodoListDTO.java
package bitc.fullstack503.moneylog_spring.dto;
import com.fasterxml.jackson.annotation.JsonProperty;
import lombok.Data;

@Data
public class TodoListDTO
{
  @JsonProperty ("todoSeq")
  private int todoSeq;
  @JsonProperty ("todoTitle")
  private String todoTitle;
  @JsonProperty ("todoStatus")
  private String todoStatus;
  @JsonProperty ("todoMemo")
  private String todoMemo;
}

File: moneylog_app/app/src/main/java/bitc/example/app/dto/TodoListDTO.kt
package bitc.example.app.dto

import com.google.gson.annotations.SerializedName

data class TodoListDTO(
  @SerializedName("todoSeq")
  var todoSeq: Int? = null,
  @SerializedName("todoTitle")
  var todoTitle: String? = null,
  @SerializedName("todoStatus")
  var todoStatus: String? = null,
  @SerializedName("todoMemo")
  var todoMemo: String? = null
)
